misc/docker/ubuntu/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:latest
+
+MAINTAINER sih4sing5hong5
+
+ENV CPU_CORE 4
+
+RUN \
+  apt-get update -qq && \
+  apt-get install -y \
+    git bzip2 wget \
+    g++ make python python3 \
+    zlib1g-dev automake autoconf libtool subversion \
+    libatlas-dev libatlas-base-dev
+
+
+WORKDIR /usr/local/
+# Use the newest kaldi version
+RUN git clone https://github.com/kaldi-asr/kaldi.git
```

Comment:
I mentioned this in the email thread a while back, but my problem with this style of Dockerfile is that it does not allow you to build Kaldi interactively. e.g., you can't modify in the C++ files or Makefiles. In order to do that, the only way that I know of is putting the Dockerfile at the top of this repo, but that's rather ugly, as well.